src/diffusers/models/transformers/transformer_2d.py

```diff
@@ -100,6 +100,7 @@ def __init__(
         attention_type: str = "default",
         caption_channels: int = None,
         interpolation_scale: float = None,
+        use_additional_conditions: bool = True,
```

Comment:
Since this argument only becomes effective when `self.config.norm_type == "ada_norm_single"`, could we throw a warning if:

```python
if use_additional_conditions and self.config.norm_type != "ada_norm_single":
```

Reply:
@lawrence-cj

Reply:
Sry. Forgot to mention here: https://github.com/huggingface/diffusers/pull/7654/commits/b32c3611993700719e505c11cc4d511f7d7f4413